db/repair_bucket.go

```diff
@@ -123,7 +123,7 @@ This is how the view is iterated:
                                                     NumProcessed: 0
 
 * It starts with an empty start key
-* For the next page, it uses the last key processed as the new start key
+* For the next pageo, it uses the last key processed as the new start key
```

Comment:
Unrelated change/typo?

Reply:
Ooh, let me try this thing...

```suggestion
* For the next page, it uses the last key processed as the new start key
```

Reply:
Nice - thanks.